controllers/order.js

```diff
@@ -24,7 +24,9 @@ const postOrder = async (req, res)=>{
 }
 const getAllOrder = async (req, res)=>{   
     // refer to graphql part
+    // why I cannot write res.status(200).send(await queryAllOrder()) to pass auto test
```

Comment:
answered in the other pull request https://github.com/konnovdev/food-order-backend/pull/27#discussion_r878186619